www/partners/texthelp.php

```diff
@@ -97,8 +121,37 @@ function $rw_toolbarLoadedCallback() {
         $rw_setStartPoint("start");
     }
 
+    var showSpeechStream = true;
+
+    function showSpeechStreamBar(){
+        showSpeechStream = !showSpeechStream;
+        window.texthelp.SpeechStream.ui.toolbar.toolbar.setVisibility(showSpeechStream);
+        $rw_stopSpeech();  
+        $rw_enableClickToSpeak(false);  
+    }
+
 </script>
 
+<style>
+    .lrn.lrn-assess .lrn-region:not(.lrn-items-region) .lrn_btn.SpeechStream_btn{
+        padding: 0.4em 0.9em 0.4em 0.7em;
+    }
+    @media all and (max-width: 1024px) and (min-width: 768px) {
+        .lrn.lrn-assess .lrn-region:not(.lrn-items-region) .lrn_btn.SpeechStream_btn{
+            padding: 0.75em 0.25em; 
+        }
+    }
+    button.lrn_btn.SpeechStream_btn:before{
+        content: '';
+        background-image: url('./speechstream-logo.png');
```

Comment:
@angelique-dubois-lrn we should move this logo to /salt-developer/code/site/demos/www/static/images and update this line.